x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_configuration.tsx

```diff
@@ -85,43 +90,130 @@ export const ConnectorConfiguration: React.FC = () => {
+                external
+              >
+                {i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.configurationConnector.connectorPackage.clientExamplesLink',
+                  { defaultMessage: 'connector client examples' }
+                )}
+              </EuiLink>
+            ),
+          }}
+        />
+      </EuiText>
+      <EuiSpacer />
+      <EuiText size="s">
+        <FormattedMessage
+          id="xpack.enterpriseSearch.content.indices.configurationConnector.connectorPackage.description.thirdParagraph"
+          defaultMessage="In this step, you will need to clone or fork the repository, and copy the generated API key and connector ID to the associated {link}. The connector ID will identify this connector to Enterprise Search."
+          values={{
+            link: (
+              <EuiLink
+                href="https://github.com/elastic/connectors-ruby/tree/main/config"
+                target="_blank"
+                external
+              >
+                {i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.configurationConnector.connectorPackage.configurationFileLink',
+                  { defaultMessage: 'configuration file' }
+                )}
+              </EuiLink>
+            ),
+          }}
+        />
+      </EuiText>
+      <EuiSpacer />
+      <EuiCodeBlock fontSize="m" paddingSize="m" color="dark" isCopyable>
+        {`${
+          apiKeyData?.encoded
+            ? `elasticsearch:
+  api_key: "${apiKeyData?.encoded}"
+`
+            : ''
+        }connector_id: "${indexData.connector.id}"
+`}
+      </EuiCodeBlock>
+      <EuiSpacer />
+      <EuiText size="s">
+        {i18n.translate(
+          'xpack.enterpriseSearch.content.indices.configurationConnector.connectorPackage.connectorDeployedText',
+          {
+            defaultMessage:
+              'Once you’ve configured the connector, deploy the connector to your self managed infrastructure.',
+          }
+        )}
+      </EuiText>
+      <EuiHorizontalRule />
+      {!indexData.connector.status || indexData.connector.status === ConnectorStatus.CREATED ? (
+        <EuiCallOut
+          title={i18n.translate(
+            'xpack.enterpriseSearch.content.indices.configurationConnector.connectorPackage.waitingForConnectorTitle',
+            {
+              defaultMessage: 'Waiting for your connector',
+            }
+          )}
+          iconType="iInCircle"
+        >
           {i18n.translate(
-            'xpack.enterpriseSearch.content.indices.configurationConnector.connectorPackage.description',
+            'xpack.enterpriseSearch.content.indices.configurationConnector.connectorPackage.waitingForConnectorText',
             {
-              defaultMessage: 'Used when configuring your connector package.',
+              defaultMessage:
+                'Your connector has not connected to Enterprise Search. Troubleshoot your configuration and refresh the page.',
```

Comment:
Shouldn't the poll pick this up?

Reply:
Only if they haven't messed up. Which they probably have if they're seeing this.